tooling/lint-package-names.sh

```diff
@@ -7,7 +7,8 @@ DEPRECATED_PACKAGES="@ciscospark/storage-adapter-session-storage \
   @ciscospark/test-helper-automation \
   @ciscospark/sparkd \
   @ciscospark/recipe-private-web-client \
-  @ciscospark/test-helper-mock-socket"
+  @ciscospark/test-helper-mock-socket \
+  @ciscospark/test-helper-server"
```

Comment:
Was this supposed to be in this PR?

Reply:
No, it's part of another, but is showing up here because it may have been excluded from the other PR. I'll double-check, but I know running this branch however I had it rebased gave me an error on this package.